examples/add_to_consensus_blacklist/main.go

```diff
@@ -0,0 +1,38 @@
+package main
```

Comment:
Seems like this file should be swapped with the examples/add_to_confiscation_tx_whitelist/main.go file